README.md

```diff
@@ -49,6 +49,19 @@ development environment for the other TDR services.
 
 When you log into the site, you will need to log in as a user from the Integration environment.
 
+#### Troubleshooting
+
+If you encounter the following error when trying to run in Intellij:
+
+`java.io.IOException: Cannot run program "npm" (in directory "/home/thomas/Tdr/tdr-transfer-frontend/npm"): error=2, No such file or directory`
+
+This is because NVM installs npm in a non-standard location.
+
+To resolve need to create a sym link with the install version of npm. Run the following commands:
```

Comment:
Maybe the start of the sentence should be "To resolve, you" or "To resolve this, you"?